content/05.main-text.md

```diff
@@ -412,7 +412,33 @@ we apply the following criteria to calculate _p~empiric~_ from summary statistic
 
 ### Rest API & backend
 
-<https://search-api.het.io/v1/>
+We created a backend application using Python's Django web framework.
+The source code is available in the [`connectivity-search-backend`](https://github.com/greenelab/connectivity-search-backend) repository.
+The primary role of the backend is manage a relational database and provide an API for requesting data.
+
+We define the database schema [using](https://github.com/greenelab/connectivity-search-backend/blob/af12f8cf2ad47d9a25ce8d1b7889390654eb3bb9/dj_hetmech_app/models.py "models.py for the dj_hetmech_app") Django's object-relational mapping framework (Figure @fig:database).
+We [import](https://github.com/greenelab/connectivity-search-backend/blob/af12f8cf2ad47d9a25ce8d1b7889390654eb3bb9/dj_hetmech_app/management/commands/populate_database.py "dj_hetmech_app populate_database management command") the data into a PostgreSQL database.
```

Comment:
Since we are describing the software a bit here - @dongbohu do we use anything PostgreSQL specific or would we expect it to work with any django-supported database? If we don't use anything specific to PostgreSQL, we could say django-supported SQL database here and then note that we use PostgreSQL.

Reply:
@cgreene The searching functionality is PostgreSQL specific.

Reply:
Let's leave it as is then 👍

Reply:
Also the JSON_Fields are a Postgres feature:

https://github.com/greenelab/connectivity-search-backend/blob/752b423a4b7b57575d66ce0b797b0a84c23267a6/dj_hetmech_app/models.py#L18